.github/workflows/e2e-test.yaml

```diff
@@ -0,0 +1,39 @@
+name: e2e docs test
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+jobs:
+  run-tests:
+    name: run tests
+    runs-on: ubuntu-latest
+    permissions:
+      contents: "read"
+      id-token: "write"
+    env:
+      PROJECT_ID: substratus-integration-tests
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: "Set up Cloud SDK"
+        uses: "google-github-actions/setup-gcloud@v1"
+        with:
+          install_components: "gke-gcloud-auth-plugin,kubectl"
+          project_id: "substratus-integration-tests"
+      - id: auth
+        name: Authenticate to Google Cloud
+        uses: google-github-actions/auth@v0.4.0
+        with:
+          workload_identity_provider: "projects/489627518739/locations/global/workloadIdentityPools/github-identities/providers/github-com"
+          service_account: "docs-test-runner@substratus-integration-tests.iam.gserviceaccount.com"
+          token_format: "access_token"
+      - name: Run tests
+        run: |
+          sudo apt-get update -y && \
+          sudo apt-get install -y --no-install-recommends \
+          python3 \
+          python3-pip \
+          gcc \
+          python3-dev && \
+          GOOGLE_CREDENTIALS=$(gcloud auth print-access-token) make test
```

Comment:
I remember gcloud would work with the credentials by default once you run the auth step. So there shouldn't be a need to set GOOGLE_CREDENTIALS

Reply:
See example here: https://github.com/google-github-actions/auth#configuring-gcloud

Reply:
Oooo I'll try that. I went through many iterations here so there's bound to be a bit left over from the various attempts

Reply:
I'll give this one more shot after the current CI run completes (may need to wait until morning since it takes a while) but I tried and failed a few more times here... I have a committed but not pushed change that shuffles the order of auth and setup which hopefully has some effect.

Reply:
Got it! Ditching the auth func